optimizely/interface.go

```diff
@@ -32,6 +33,7 @@ type ProjectConfig interface {
 	GetEventByKey(string) (entities.Event, error)
 	GetExperimentByKey(string) (entities.Experiment, error)
 	GetFeatureByKey(string) (entities.Feature, error)
+	GetFeatureFlagByKey(string) (datafileEntities.FeatureFlag, error)
```

Comment:
This is wrong. Feature flags an d features are the same thing. You are bringing in the data file raw feature flags in here, which is not what we want.